articles/ros2_threat_model.md

```diff
@@ -2097,11 +2096,13 @@ computer.
   * [`raspicam2_node`][raspicam2_node] a node publishing Raspberry Pi Camera
     Module data to ROS 2.
 * An XRCE Agent runs on the Raspberry, and is used by a DDS-XRCE client running
-  on the [OpenCR 1.0 board][opencr_1_0], that publishes  IMU sensor data to ROS
+  on the [OpenCR 1.0 board][opencr_1_0], that publishes IMU sensor data to ROS
   topics, and controls the wheels of the TurtleBot based on the
-  [teleoperation][tb3_teleop] messages published as ROS topics.
+  [teleoperation][tb3_teleop] messages published as ROS topics. This channel uses
+  serial communication.
 * A Lidar driver process running on the Raspberry interfaces with the Lidar, and
-  uses a DDS-XRCE client to publish the sensor data to several ROS topics.
+  uses a DDS-XRCE client to publish the sensor data to several ROS topics. This
+  channel uses udp.
```

Comment:
udp -> UDP

Reply:
Also, I'm not clear on how it works, does this node actually use UDP over the wire or is just between the lidar driver and the XRCE agent (on localhost)?

Reply:
You're right, there is a drive on the raspberry pi that interacts with the LIDAR sensor. The XRCE agent and client are both running on the raspberry pi. I'll try to clarify that